README.rst

```diff
@@ -136,6 +144,14 @@ Run quality assurance tests with::
 
     ./run-qa-checks
 
+You can run all unit tests by launching the dedicated script::
```

Comment:
so, I tried this part and here's a few issues I found:

./install-dev.sh needs sudo, I think it doesn't hurt to use `sudo ./install-dev.sh ` in the example.

Then, run-qa-checks depends on openwisp-utils, but there's no mention of this, so the script will fail if a user follows this verbatim. Please fix this.

Whenever you write a piece of docs,if you really want other people working with you to be able to make something useful out of it, try to test it by following verbatim on a new system.